devtools/conda-envs/test_env.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
   - openmm
   - openff-toolkit >=0.11
```

Comment:
Apologies, at least some of the test errors you're seeing are from a release we just cut (#365). This will fix that issue. 
```suggestion
  - openff-toolkit >=0.11
  - openff-interchange =0.4.1
```